ckanext/charts/theme/mixins.scss

```diff
@@ -0,0 +1,32 @@
+.date-formats-container {
+
+    &__row {
```

Comment:
I suggest avoiding using this -> `&__row`, cause it's hard to find by searching for a full class name in a project. We're not saving anything here, cause it will be translated into CSS with a full class name anyway.

Reply:
anyway, if you'll use a table, you probably won't need any styles